logs/parse_test.go

```diff
@@ -10,7 +10,25 @@ import (
 	"github.com/pganalyze/collector/state"
 )
 
-var parse2Tests = []parseTestpair{
+type parseTestpair struct {
+	prefixIn  string
+	lineIn    string
+	lineInTz  *time.Location
+	lineOut   state.LogLine
+	lineOutOk bool
+}
+
+func mustTimeLocation(tzStr string) *time.Location {
+	tz, err := time.LoadLocation(tzStr)
+	if err != nil {
+		panic(err)
+	}
+	return tz
+}
+
+var BSTTimeLocation = mustTimeLocation("Europe/London")
```

Comment:
These are all copied from `legacy_parse_test.go`. They were shared by this file before.